web/test/pages/onboarding/onboarding-shared.test.tsx

```diff
@@ -112,6 +113,23 @@ describe("onboarding - shared", () => {
     expect(currentBusiness().profileData.businessPersona).toEqual("STARTING");
     expect(currentBusiness().profileData.industryId).toEqual(industry);
     expect(currentBusiness().onboardingFormProgress).toEqual("COMPLETED");
+    expect(currentBusiness().profileData.operatingPhase).toEqual(OperatingPhaseId.GUEST_MODE);
+  });
+
+  it("routes to dashboard page when sector question is set by using sector query string", async () => {
```

Comment:
[sand] I might try to rename this slightly. Something indicating that we are completing onboarding as an up-and-running business with the sector indicated via the query string.

Reply:
I'll update the title